tests/unit/bootstrap.ts

```diff
@@ -3,6 +3,13 @@ import * as chaiEnzyme from "chai-enzyme";
 
 chai.use(chaiEnzyme());
 
+window.matchMedia = window.matchMedia || (() => {
```

Comment:
if(!window.matchMedia) { .... }
And this code must be in standalone file.
I suppose we should use real polyfill:
https://github.com/paulirish/matchMedia.js/